administrator/components/com_scheduler/src/Table/TaskTable.php

```diff
@@ -167,6 +167,44 @@ protected function _getAssetName(): string
         return 'com_scheduler.task.' . (int) $this->$k;
     }
 
+    /**
+     * Method to return the title to use for the asset table.
+     *
+     * @return  string
+     *
+     * @since   5.0.2
+     */
+    protected function _getAssetTitle(): string
+    {
+        return $this->title;
+    }
+    
+    /**
+     * Method to get the parent asset under which to register this one.
+     * By default, all assets are registered to the ROOT node with ID,
+     * which will default to 1 if none exists.
+     * The extended class can define a table and id to lookup.  If the
+     * asset does not exist it will be created.
+     *
+     * @param   Table|null  $table  A Table object for the asset parent.
+     * @param   null        $id     Id to look up
+     *
+     * @return  integer
+     *
+     * @since   5.0.2
```

Comment:
```suggestion
     * @since   __DEPLOY_SINCE__
```

This will get the correct version number when it is released